src/app/screens/BookDetail/components/BookInfo/index.tsx

```diff
@@ -74,16 +75,22 @@ function BookInfo() {
         <Button buttonStyle={styles.watchlistButton} buttonTextStyle={styles.watchlistButtonText}>
           ADD TO WATCHLIST
         </Button>
-        <AnimatedTouchableOpacity
-          activeOpacity={1}
-          style={[styles.rentButton, rentButtonAnimatedStyle]}
-          onPress={handleRent}>
-          {bookRented ? (
-            <Image style={styles.rentedIcond} source={rentedIcond} />
-          ) : (
-            <Text style={styles.rentButtonText}>RENT</Text>
-          )}
-        </AnimatedTouchableOpacity>
+        <LinearGradient
+          style={styles.rentButtonWrapper}
+          start={{ x: 0, y: 0 }}
```

Comment:
Are the start and end props necessary? what happens if you remove them?

Reply:
Works well

Reply:
Can you remove the start prop too?

Reply:
Yes